addon/services/media.js

```diff
@@ -77,35 +77,27 @@ import Evented from '@ember/object/evented';
 * @class     Media
 * @extends   Ember.Object
 */
-export default Service.extend(Evented, {
-  // eslint-disable-next-line ember/no-ember-testing-in-module-scope
-  _mocked: Ember.testing,
-  _mockedBreakpoint: 'desktop',
```

Comment:
https://github.com/ember-cli/eslint-plugin-ember/pull/272/files/36b68e3e8d9ffef76ff1bcfac9cd8154af316ee4

> Since ember-cli-qunit@4.1.0 / ember-qunit@3.0.0, `Ember.testing` is only set to
true while a test is executing instead of all the time.

Reply:
Previous addition 

https://github.com/freshbooks/ember-responsive/pull/124

> During tests, the media service will be in "desktop" unless told otherwise.

Basically this is only true initially then Ember.testing is set to `false`